charts/mimir-distributed/templates/rolebinding.yaml

```diff
@@ -12,4 +12,5 @@ roleRef:
 subjects:
```

Comment:
Can't comment on line 1 but the if condition seems to be missing the `.Values.rbac.sccEnabled`?

```
{{- if or .Values.rbac.pspEnabled .Values.rbac.sccEnabled }}
```